### PR TITLE
Document correct db transaction handling

### PR DESCRIPTION
**Pull request by ChristophWurst**
Bildschirmfoto von 2021-08-13 11-27-21

### CODE REVIEW

File: developer_manual/basics/storage/database.rst [hunk 1]
```diff
@@ -45,6 +45,33 @@ Inside your database layer class you can now start running queries like:
     }
 
 
+Transactions
+------------
+
+Database operation can be run in a nested transaction to commit or roll back a group of changes in an atomic fashion.
```
Comment (marcelklehr) [inline developer_manual/basics/storage/database.rst:51]: operation**s**

What is the significance of *nested*? Can I put a transaction into another transaction?

Comment (ChristophWurst) [reply developer_manual/basics/storage/database.rst:51]: > Can I put a transaction into another transaction?

Yes. You can have multiple levels.

Comment (ChristophWurst) [reply developer_manual/basics/storage/database.rst:51]: But I will drop this. It's unnecessarily confusing :)

**Review by marcelklehr — APPROVED** [review]
:+1:

**Pull request merged**